spark/src/main/java/org/apache/iceberg/spark/SparkReadOptions.java

```diff
@@ -50,4 +50,10 @@ private SparkReadOptions() {
 
   // Set ID that is used to fetch file scan tasks
   public static final String FILE_SCAN_TASK_SET_ID = "file-scan-task-set-id";
+
+  // skip snapshots of type delete while reading stream out of iceberg table
+  public static final String READ_STREAM_SKIP_DELETE = "read-stream-skip-delete";
+
+  // skip snapshots of type replace while reading stream out of iceberg table
+  public static final String READ_STREAM_SKIP_REPLACE = "read-stream-skip-replace";
```

Comment:
Why make this optional? Since replace commits don't affect the logical data in the table, shouldn't they always be skipped?

Reply:
That makes sense. Added an explicit option based off of my interpretation of https://github.com/apache/iceberg/pull/2660#discussion_r658351562
But maybe an explicit option was not the intention of the comment? :)

Reply:
No, I wasn't implying that there should be an option. I was saying they should just be skipped.